src/index.ts

```diff
@@ -165,6 +200,99 @@ const createCondaMetaFile = (
   }
 };
 
+export const bootstrapFromEmpackPackedEnvironment = async (
+  packagesJsonUrl: string,
+  verbose: boolean = true,
+  skipLoadingSharedLibs: boolean = false,
+  Module: any
+) => {
+  if (verbose) {
+    console.log('fetching packages.json from', packagesJsonUrl);
+  }
+
+  // fetch json with list of all packages
+  let empackEnvMeta = await fetchJson(packagesJsonUrl);
+  let allPackages: IEmpackEnvMetaPkg[] = empackEnvMeta.packages;
+  let prefix = empackEnvMeta.prefix;
+
+  console.log('allPackages', allPackages);
+
+  if (verbose) {
+    console.log('makeDirs');
+  }
+  //Module.create_directories("/package_tarballs");
+
+  // enusre there is python and split it from the rest
+  if (verbose) {
+    console.log('splitPackages');
+  }
+  let splitted: ISplittedPackages = splitPackages(allPackages);
```

Comment:
this splitting in python and not python packages should not be needed.
We needed this because we wanted to download python first, and then untar everything with python itself once thats donwloaded.

But this should not be needed with the untar-ing done by untarjs and not python